pkg/scd/store/cockroach/constraints.go

```diff
@@ -203,7 +228,8 @@ func (c *repo) SearchConstraints(ctx context.Context, v4d *dssmodels.Volume4D) (
 				COALESCE(starts_at <= $3, true)
 			AND
 				COALESCE(ends_at >= $2, true)
-			LIMIT $4`, constraintFieldsWithoutPrefix)
+			LIMIT $4;
+			`, constraintFieldsWithoutPrefix)
```

Comment:
Is setting a final `;` required for yugabyte? If so, why is it not the case of query on scd_constraints and availabilities above? And if not, why adding it here?